posts/2022-07-02-sunny-tech-en.adoc

```diff
@@ -0,0 +1,227 @@
+
+https://twitter.com/mathishammel[Mathis Hammel] presented us his home-made solution to detect *deep fakes*.
+The site https://thispersondoesnotexist.com[thispersondoesnotexist] generates fake faces for free.
+
+.Fake face recovered from thispersondoesnotexist
+image::fake-face.jpeg[Fake face recovered from, width = 800]
+
+Ill-intentioned individuals use these services to create fake profiles en masse on social networks.
+To remedy this, Mathis created a solution to detect them from scratch.
+
+.Summary of the presentation by @ptibulle
+image::deep-fake.jpeg[Summary of the presentation by @ptibulle, width = 800]
+
+The speaker started his reflection with a simple but effective solution.
+Before explaining it, we need to go back to the methods used to realize this AI.
+
+The artificial intelligence technique used to generate the images is a https://en.wikipedia.org/wiki/Generative_adversarial_network[GAN] (Generative adversarial network
+).
+This type of model is based on three parts:
+
+1. a *generator* that will produce random images at the beginning and then more and more realistic as you train
+2. a bank of real images
+3. a *discriminator* that will have to detect if the image presented to it comes from the generator or from the image bank
+
+The goal of the generator is to produce the most realistic image possible to deceive the discriminator and the goal of the discriminator is to detect as well as possible the false images generated.
+After many training cycles, the two modules will improve and become more and more efficient.
+
+The first trick to detect false images generated by this model comes from the *image bank used*.
+Indeed, the faces coming from this group of photos have a particularity: _the eyes are always placed in the same place_.
+Therefore, the AI will try to create images as close as possible to the real ones and will therefore also place the eyes in the same place.
+The most naive solution is to look at the position of the eyes on the image and if it matches the known position, the image is false.
+But this solution is not very robust to changes in the pictures, so the presenter developed a more advanced method.
+
+To do this, it has set up a service that retrieves and stores images from the site_thispersondoesnotexist_.
+It has made several optimizations in order not to store the raw images on the server, as this would take up too much disk space.
+Then, when you want to detect a fake face, the tool will compare with its database and according to a score, send back yes or no.
+If you want to test, Mathis hosts a https://fada.h25.io/[website] allowing to detect fake images directly from the browser.
+Moreover, this web site displays interesting information such as look-alikes, when the fake image was generated, or the eyes technique explained before.
+
+This talk is really well popularized and we are directly captured by the subject thanks to a very good speaker!
```

Comment:
i'm not 100% clear what you're trying to say here - do you mean the talk was about a popular subject? Or that it was clearly explained? Sorry, i'm not quite getting this :(